code/__defines/subsystems.dm

```diff
@@ -3,9 +3,9 @@
 #define INITIALIZATION_INNEW_MAPLOAD  2	//New should call Initialize(TRUE)
 #define INITIALIZATION_INNEW_REGULAR  3	//New should call Initialize(FALSE)
 
-#define INITIALIZE_HINT_NORMAL   0  //Nothing happens
-#define INITIALIZE_HINT_LATELOAD 1  //Call LateInitialize
-#define INITIALIZE_HINT_QDEL     2  //Call qdel on the atom
+var/INITIALIZE_HINT_NORMAL   = new/datum/init_hint/normal   //Nothing happens
```

Comment:
Nitpick, but `new /foo` (with a space) makes it easier to search for uses of `new`.